nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -237,16 +243,24 @@ WaypointFollower::followWaypoints()
       RCLCPP_INFO(
         get_logger(), "Task execution at waypoint %i %s", goal_index,
         is_task_executed ? "succeeded" : "failed!");
+      if (!is_task_executed) {
```

Comment:
Remove this - there would be no error code because this is for the waypoint executor plugin, not from the constituent navigation server failures. We haven't defined a set of exceptions/error codes for the waypoint task executors. Arguably, we cannot since its so broad. 

Unless we added an error code for WPF for failed task executor just generically. That would be smart.

Reply:
Sounds like a future project :).

Reply:
Its just adding an enum to the waypoint follower for the next server up (6000's?) for `TASK_EXECUTOR_FAILED`

Reply:
Is the next number in line actually 6000?

Reply:
100-199 controller
200-299 compute path to pose 
300-399 compute path through poses 
400-499 smoother server? 

So, it depends on if there is anything else we want to add.

Reply:
OK - leave it as 600, I have something in mind between them for early next year